usim.py

```diff
@@ -384,7 +384,8 @@ def monitor_sub(q, index):
 	t0 = time.time()
 	log('Started main thread %d' % (index+1))
 	while True:
-		log("Restarting loop", console_only=True)
+		#log("Restarting loop", console_only=True)
```

Comment:
This seems like a very good change, but why commit commented code?

Maybe I'm missing something here (and it's trambelus's call, really), but doesn't [a VCS kind of eliminate the need to do this?](http://programmers.stackexchange.com/a/190222/186238)

pbuf